pkg/cmd/registry/artifact/crud/create/create.go

```diff
@@ -83,17 +56,11 @@ func NewCreateCommand(f *factory.Factory) *cobra.Command {
 	}
 
 	cmd := &cobra.Command{
-		Use:   "create",
-		Short: "Creates new artifact from file or standard input",
-		Long:  longDescription,
-		Example: `
-# Create an artifact in default group
-rhoas service-registry artifact create my-artifact.json
-
-# Create an artifact with specified type
-rhoas service-registry artifact create --type=JSON my-artifact.json
-		`,
-		Args: cobra.RangeArgs(0, 1),
+		Use:     f.Localizer.MustLocalize("artifact.cmd.create.use"),
```

Comment:
I did not internationalise use but I think we aren'tp consistent in this across cli. 
Command names should not change

Reply:
Let me know if you want to revert strings for "use".

Reply:
No need. All good.